### PR TITLE
Minimum viable product.

### PR DESCRIPTION
**Pull request by mfletchernyc**
image

I think the MTA's [bus app](https://bustime.mta.info) is kind of ugly, so I'm making my own. See it working [here](http://fletcher.nyc/etc/bus-tracker/).

The app shows the buses, stops and routes. Nothing moves yet. Preliminary!

### CODE REVIEW

File: src/Tracker.test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// import { render, screen } from '@testing-library/react'
+// import App from './App'
+
+describe('No tests.', () => {
+  it('needs tests', () => {
+    // It does...
```
Comment (kayernyc) [inline src/Tracker.test.js:6]: truth

**Review by kayernyc — APPROVED** [review]
The gitignore looks a little light, and the {{thing}} was new to me!

👍

**Comment by mfletchernyc** [comment]
> The gitignore looks a little light, and the {{thing}} was new to me!
> 
> 👍

Thanks, @kayernyc! What are your recommendations for `.gitignore`? Also what is `{{thing}}`?

File: src/Tracker.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        rotationOrigin="center center"
+      />
+    )
+  }
+
+  const drawRoute = (line) => (
+    <GeoJSON
+      data={line.route}
+      key={uniqid()}
+      style={{ color: `#${line.color}33` }}
```
Comment (kayernyc) [inline src/Tracker.js:78]: {{ thing }}

**Review by kayernyc — COMMENTED** [review]
marked a {{ thing }}

**Comment by kayernyc** [comment]
I often start with [this](https://gist.github.com/andreasonny83/b24e38b7772a3ea362d8e8d238d5a7bc) It has more than you need, but if you ever get collaborators who use a different setup of the same IDE you'll be happy you ignored their settings :)

File: src/Tracker.js (hunk 2 above)
Comment (mfletchernyc) [reply src/Tracker.js:78]: Ah right. JS + React style object. It looked like your internal templating language didn't render properly. 😆

**Comment by mfletchernyc** [comment]
> I often start with [this](https://gist.github.com/andreasonny83/b24e38b7772a3ea362d8e8d238d5a7bc)...

Nice! Added any rules from there not already in `.gitignore`.

**Pull request merged**